packages/vaex-ml/vaex/ml/transformations.py

```diff
@@ -226,8 +226,16 @@ def fit(self, df):
         for i in self.features:
             expression = _ensure_strings_from_expressions(i)
             unique = df.unique(expression)
-            unique = np.sort(unique)  # this can/should be optimized with @delay
-            uniques.append(unique.tolist())
+            unique = vaex.array_types.tolist(unique)
+
+            if None in unique:
+                unique = list(filter(None, unique))
```

Comment:
this is dangerous, for instance `list(filter(None, [0, 1, 2]))` will tell you why. Surprised the tests didn't pick that up.

Reply:
unique.remove(None) would also work